pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -2955,7 +2956,7 @@ public CompactedTopic getCompactedTopic() {
 
     @Override
     public boolean isSystemTopic() {
-        return false;
+        return HEARTBEAT_TENANT.equals(TopicName.get(topic).getTenant());
```

Comment:
Not like this. There are some other system topics.
The origin check seems to be `org.apache.pulsar.broker.service.BrokerService#isSystemTopic`.

Reply:
#13495 made changes on `getReplicationClusters`  don't need this change any more